lib/autsr.gi

```diff
@@ -13,6 +13,9 @@
 ##    Automorphism group computation and isomorphism testing in finite groups.
 ##    J. Symb. Comput. 35, No. 3, 241-267 (2003)
 
+# The checker complains about a used variable being unused.
+# gaplint: disable = analyse-lvars
```

Comment:
Looks to me that the variable scharorb is assigned twice but not used, and that the linter is reporting this correctly.